### PR TITLE
CLI begin end block

### PR DESCRIPTION
**Pull request by ebuchman**

### CODE REVIEW

File: cmd/abci-cli/abci-cli.go [hunk 1]
```diff
@@ -295,6 +311,68 @@ func cmdCommit(c *cli.Context) error {
 	return nil
 }
 
+// Delineate beginning of block with header and hash
+func cmdBeginBlock(c *cli.Context) error {
+	args := c.Args()
+	if len(args) != 2 {
+		return errors.New("Command begin block takes 2 arguments: hash and header")
```
Comment (melekes) [inline cmd/abci-cli/abci-cli.go]: `header and hash` (wrong order)

**Comment by adrianbrink** [comment]
@ebuchman Can this be merged?

**Comment by ebuchman** [comment]
We need to write a test for it.

**Comment by adrianbrink** [comment]
docs: https://github.com/tendermint/tendermint/pull/1061

File: tests/test_cli/ex1.abci [hunk 2]
```diff
@@ -4,7 +4,9 @@ commit
 deliver_tx "abc"
 info
 commit
+endblock 0
 query "abc"
 deliver_tx "def=xyz"
 commit
 query "def"
+endblock 1
```
Comment (zramsay) [inline tests/test_cli/ex1.abci]: `end_block` ?
where's `begin_block`?

**Comment by ebuchman** [comment]
Maybe this was ill conceived. Sorry for wasting anyone's time :)

**Pull request closed without merging**